bzip2-sys/build.rs

```diff
@@ -41,6 +41,10 @@ fn main() {
         }
     }
 
+    if cfg!(feature = "thin") {
+        cfg.opt_level_str("z");
+    }
```

Comment:
This should probably forward the `OPT_LEVEL` env var instead of adding a feature.